elastic/beats
# Docker health metricset cleanup

* Update data.json with latest data
* Clean up variable naming
* Add system test to also check docs
* Introduce healthcheck in dockerfile to allow automated testing
* Update CHANGELOG
* Rename wrongly named file
